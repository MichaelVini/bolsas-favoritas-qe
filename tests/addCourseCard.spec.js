import { shallowMount } from '@vue/test-utils'
import AddCouseCard from './../src/components/AddCourseCard/AddCourseCard.vue'


describe('AddCouseCard.vue', ()=> {
    it('should render default state', () => {
        const wrapper = shallowMount(AddCouseCard)
        expect(wrapper.html()).toMatchSnapshot()
    })
})