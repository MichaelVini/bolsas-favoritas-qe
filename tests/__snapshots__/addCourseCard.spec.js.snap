// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddCouseCard.vue should render default state 1`] = `
"<div class=\\"add-courses\\"><img src=\\"@/assets/plus-circle.svg\\" alt=\\"\\" class=\\"add-courses__image\\">
  <h2>Adicionar bolsa</h2>
  <p class=\\"add-courses__text\\">Clique para adicionar bolsas de cursos do seu interesse</p>
</div>"
`;
